GeneratorAnalysis/Scripts/GeneratorInteBreakDown.cpp

```diff
@@ -426,7 +451,7 @@ void GeneratorInteBreakDown() {
                     textSlice->DrawLatexNDC(0.2, 0.81, Labels[iSample] + "      " + LatexLabel[ReducedPlotName].ReplaceAll("All events","") + SliceLabel);
 
                     gPad->RedrawAxis();
-                    TString dir = "/exp/sbnd/app/users/" + (TString)UserName + "/CC1muAnalysis";
+                    TString dir = "/exp/sbnd/app/users/" + (TString)UserName + "/BuildEventGenerators/CC1muAnalysis";
```

Comment:
Can we make it so that this remains the same in the repo? (so maybe either move your local repo so it not inside `BuildEventGenerators/`, or avoid committing changes that modify this `dir`; same comment for all instances of the same change)